diff -Naur a/lang/python/hu/LC_MESSAGES/python.po b/lang/python/hu/LC_MESSAGES/python.po
--- a/lang/python/hu/LC_MESSAGES/python.po	2022-05-29 17:04:24.000000000 +0200
+++ b/lang/python/hu/LC_MESSAGES/python.po	2022-06-11 16:30:51.709048595 +0200
@@ -119,11 +119,11 @@
 
 #: src/modules/unpackfs/main.py:34
 msgid "Filling up filesystems."
-msgstr "Fájlrendszerek betöltése."
+msgstr "Fájlrendszerek átmásolása."
 
 #: src/modules/unpackfs/main.py:254
 msgid "rsync failed with error code {}."
-msgstr "az rsync elhalt a(z) {} hibakóddal"
+msgstr "Az rsync elhalt a(z) {} hibakóddal"
 
 #: src/modules/unpackfs/main.py:299
 msgid "Unpacking image {}/{}, file {}/{}"
@@ -139,7 +139,7 @@
 
 #: src/modules/unpackfs/main.py:430
 msgid "No mount point for root partition"
-msgstr "Nincs betöltési pont a root partíciónál"
+msgstr "Nincs csatolási pont a root partícióhoz"
 
 #: src/modules/unpackfs/main.py:431
 msgid "globalstorage does not contain a \"rootMountPoint\" key."
@@ -147,7 +147,7 @@
 
 #: src/modules/unpackfs/main.py:434
 msgid "Bad mount point for root partition"
-msgstr "Rossz betöltési pont a root partíciónál"
+msgstr "Rossz csatolási pont a root partícióhoz"
 
 #: src/modules/unpackfs/main.py:435
 msgid "rootMountPoint is \"{}\", which does not exist."
@@ -176,6 +176,7 @@
 "Failed to find unsquashfs, make sure you have the squashfs-tools package "
 "installed."
 msgstr ""
+"Az unsquashfs nem található, a squashfs-tools csomag telepítve van?"
 
 #: src/modules/unpackfs/main.py:481
 msgid "The destination \"{}\" in the target system is not a directory"
@@ -188,7 +189,7 @@
 #: src/modules/services-systemd/main.py:59
 #: src/modules/services-openrc/main.py:93
 msgid "Cannot modify service"
-msgstr "a szolgáltatást nem lehet módosítani"
+msgstr "A szolgáltatást nem lehet módosítani"
 
 #: src/modules/services-systemd/main.py:60
 msgid ""
@@ -233,7 +234,7 @@
 #: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
 #: src/modules/dummypython/main.py:94
 msgid "Dummy python step {}"
-msgstr "Hamis {}. Python lépés"
+msgstr "Hamis Python lépés: {}"
 
 #: src/modules/bootloader/main.py:43
 msgid "Install bootloader."
@@ -255,7 +256,7 @@
 
 #: src/modules/localecfg/main.py:31
 msgid "Configuring locales."
-msgstr "nyelvi értékek konfigurálása."
+msgstr "Nyelvi beállítások."
 
 #: src/modules/mount/main.py:42
 msgid "Mounting partitions."
@@ -344,11 +345,11 @@
 
 #: src/modules/dracut/main.py:27
 msgid "Creating initramfs with dracut."
-msgstr "initramfs létrehozása ezzel: dracut."
+msgstr "initramfs létrehozása a dracut programmal."
 
 #: src/modules/dracut/main.py:49
 msgid "Failed to run dracut on the target"
-msgstr "dracut futtatása nem sikerült a célon."
+msgstr "A dracut futtatása nem sikerült a célon."
 
 #: src/modules/grubcfg/main.py:28
 msgid "Configure GRUB."
@@ -396,21 +397,22 @@
 
 #: src/modules/displaymanager/main.py:992
 msgid "No display managers selected for the displaymanager module."
-msgstr "Nincs kijelzőkezelő kiválasztva a kijelzőkezelő modulhoz."
+msgstr "Nincs kiválasztott grafikus beléptetö a megfelelő modulhoz."
 
 #: src/modules/displaymanager/main.py:993
 msgid ""
 "The displaymanagers list is empty or undefined in both globalstorage and "
 "displaymanager.conf."
-msgstr ""
+msgstr "A beléptetők listája üres, vagy nincs megadva a sem globalstorage-ben sem"
+" a displaymanager.conf fájlban."
 
 #: src/modules/displaymanager/main.py:1075
 msgid "Display manager configuration was incomplete"
-msgstr "A kijelzőkezelő konfigurációja hiányos volt"
+msgstr "A grafikus beléptető konfigurációja hiányos volt"
 
 #: src/modules/initramfscfg/main.py:32
 msgid "Configuring initramfs."
-msgstr "initramfs konfigurálása."
+msgstr "Az initramfs konfigurálása."
 
 #: src/modules/hwclock/main.py:26
 msgid "Setting hardware clock."
